account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/UpdatePhoneNumberHandler.java

```diff
@@ -17,11 +17,8 @@
 import uk.gov.di.accountmanagement.services.CodeStorageService;
 import uk.gov.di.authentication.shared.entity.ErrorResponse;
 import uk.gov.di.authentication.shared.exceptions.UserNotFoundException;
-import uk.gov.di.authentication.shared.helpers.ClientSubjectHelper;
-import uk.gov.di.authentication.shared.helpers.IpAddressHelper;
+import uk.gov.di.authentication.shared.helpers.*;
```

Comment:
Just a fly-by review here - haven't looked in detail. We should avoid wildcard imports - see [here](https://gds-way.digital.cabinet-office.gov.uk/manuals/programming-languages/java.html#imports) for explanation and a way of configuring your editor to avoid it doing this automatically